doc/dependencies.rst

```diff
@@ -12,7 +12,7 @@ Security aspects
 ----------------
 
 A project that uses Hunter first downloads the hunter package itself. This is
-done with a help of HunterGate CMake module. A copy of the HunterGate module
+done with the help of HunterGate CMake module. A copy of the HunterGate module
```

Comment:
Can you add "the" between "of HunterGate" i.e. "done with the help of the HunterGate CMake module."